modules/images-configuration-allowed.adoc

```diff
@@ -59,7 +59,7 @@ status:
 Either the `allowedRegistries` parameter or the `blockedRegistries` parameter can be set, but not both.
 ====
 +
-The Machine Config Operator (MCO) watches the `image.config.openshift.io/cluster` resource for any changes to the registries. When the MCO detects a change, it drains the nodes, applies the change, and uncordons the nodes. After the nodes return to the `Ready` state, the allowed registries list is used to update the image signature policy in the `/etc/containers/policy.json` file on each node.
+The Machine Config Operator (MCO) watches the `image.config.openshift.io/cluster` resource for any changes to the registries. When the MCO detects a change, it triggeres mcp rollout and the allowed registries list is used to update the image signature policy in the `/etc/containers/policy.json` file on each node.
```

Comment:
What is "mcp rollout"? This seems like an abbreviation/jargon that we would want to use the full terminology for in docs.